alcs-frontend/src/app/shared/meeting-overview/meeting-overview.component.ts

```diff
@@ -36,21 +38,32 @@ export class MeetingOverviewComponent implements OnInit, OnDestroy {
   private meetings: UpcomingMeetingBoardMapDto | undefined;
   customDateFormat = 'ddd YYYY-MM-DD';
   searchText = '';
+  isCommissioner = false;
 
   viewData: BoardWithDecisionMeetings[] = [];
 
   constructor(
     private meetingService: DecisionMeetingService,
     private boardService: BoardService,
     private applicationDocumentService: ApplicationDocumentService,
-    private toastService: ToastService
+    private toastService: ToastService,
+    private authService: AuthenticationService
   ) {}
 
   ngOnInit(): void {
     this.boardService.$boards.pipe(takeUntil(this.destroy)).subscribe((boards) => {
       this.boards = boards;
       this.loadMeetings();
     });
+
+    this.authService.$currentUser.subscribe((currentUser) => {
+      if (currentUser) {
+        this.isCommissioner =
+          currentUser.client_roles && currentUser.client_roles.length === 1
```

Comment:
I wonder if this should simple check for commissioner role presence rather then only for one role that commisioner

Reply:
Nevermind, I saw the code bellow